webapp/src/websocket/index.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+import ActionTypes from '../action_types';
+import Constants from '../constants';
+import {getConnected, notifyCanStartMeeting} from '../actions';
+
+export function handleWebexConnected(store) {
+    
+    return (msg) => {
+        if (!msg.data) {
+            return;
+        }
+
+        if (msg.data.success) {
+          getConnected()(store.dispatch, store.getState);
```

Comment:
this is better called like:
```js
store.dispatch(getConnected());
store.dispatch(notifyCanStartMeeting());
```

Reply:
Updated, thank you for the correction of proper usage. This has been really helpful for me.